Capri.Web/src/store/modules/exampleModule.ts

```diff
@@ -1,16 +1,16 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
 
-//example template of a module to show how it works
-//remove it or replace with your own data
+// example template of a module to show how it works
+// remove it or replace with your own data
 export const ExampleModule = {
     state: {
-
     },
-    mutations: {    //synchronous change
+    // tslint:disable-next-line: object-literal-sort-keys
```

Comment:
You can remove that because it's added to tslint config.

Reply:
Removed.

Reply:
Removed.